README.md

```diff
@@ -13,12 +13,12 @@ Confidential Computing is the protection of data in use by performing computatio
 The purpose of the Confidential Computing Consortium is to raise,
 budget and spend funds in support of various open source and/or open standards projects
 relating to:
-* defining confidential computing and accelerate acceptance and adoption in the
+* defining confidential computing and accelerating acceptance and adoption in the
 market.
-*  developing open enterprise-grade building blocks (e.g. open specifications and open
-source licensed projects), including a confidential managed compute portfolio and
-development experience, to enable easy development and management of enterprisegrade confidential compute applications, and
-* defining foundational services and frameworks that are confidential-aware and
+* developing open enterprise-grade building blocks (e.g., open specifications and open
+source licensed projects), including a confidential computing portfolio and
+development experience, to enable easy development and management of enterprise-grade confidential computing applications, and
+* defining foundational services and frameworks that are confidential-computing aware and
 minimize the need for trust.
 
```

Comment:
Add something around "raising industry awareness of Confidential Computing and use cases across various sectors"?  Also, maybe regulators and standards body engagement?  If so, this stuff needs to go to the GB.

Reply:
This text comes from the charter (grammatical errors and all). I'm fine cleaning it up here but I agree that any substantive rewrite is a board action outside the scope of this PR.

Reply:
Yeah, https://confidentialcomputing.io/wp-content/uploads/sites/10/2023/03/Confidential-Computing-Consortium-Charter.pdf Exhibit B, point 1(a) has the same issues.  Dan can you raise this PR with the board as a charter amendment to vote on?